src/eq_fault_geom/geomio/cfm_shp_to_vtk.py

```diff
@@ -0,0 +1,214 @@
+from pathlib2 import Path
+import string
+import geopandas as gpd
+import pandas as pd
+from shapely.geometry import LineString
+import numpy as np
+import meshio
+# import pdb
+# pdb.set_trace()
+
+# Epsilon value.
+eps = 5000.0
+
+# Cell type.
+cell_type = 'triangle'
+# cell_type = 'quad'
+
+# Output directory and file suffix.
+output_dir = "../../../data/cfm_shapefile/cfm_vtk"
+vtk_suffix = ".vtk"
+
+# Create output directory if it does not exist.
+output_path = Path(output_dir)
+output_path.mkdir(parents=True, exist_ok=True)
+
+# Which dip and depth values to use.
+dip_use = 'Dip_Best'
+depth_use = 'Depth_Max'
+
+# Dictionary for replacing numbers with letters.
```

Comment:
Feedback from Russ et al was 'please don't change the fault identitifier' . So I think we'll just live with the minor Paraview annoyave caused by the numeric suffixes

Reply:
I no longer replace numbers with letters; however, I'm still replacing spaces with underscores. Should I leave in the spaces?  I hate spaces in filenames.

Reply:
Sounds like replacing spaces with underscores is OK.